### ISSUE TITLE
Phillipians 2:13 is wrong

### ISSUE THREAD

**Issue by adisciple50** [openenglishbible/Open-English-Bible#322]
the NIV is correct.

**Comment by countingpine** [comment]
I don't want to give an opinion on whether I agree or disagree, but I want this to be a helpful and constructive report, so it can be more easily remedied or closed.

https://github.com/openenglishbible/Open-English-Bible/blob/2020.2/source/50-Philippians.usfm.db#L68
"Remember it is God who, in his kindness, is at work within you, enabling you both to will and to work."

[NIV 2011](https://www.biblegateway.com/passage/?search=Philippians%202%3A13&version=NIV): for it is God who works in you to will and to act in order to fulfill his good purpose.
(NIV 1984 says: "according to his good purpose")

To map the components of the translations together:
1. "Remember it is God": "for it is God"
2. "who, in his kindness": "who ... in order to fulfill his good purpose."
3. "is at work within you, enabling you": "works in you"
4. "both to will and to work": "to will and to act"

(It would potentially be more helpful, but harder for me, to go to the original Greek instead. But the reporter evidently deems the NIV to carry sufficient authority.)

In my opinion, the two components that differ most is:
- in 2, moving the end of the statement and changing "according to his good purpose" to "in his kindness".
- in 3, the addition of "enabling you" - the NIV perhaps suggest God wills and works in us, the OEB perhaps suggests that we will and work thanks to God's enabling.  (I don't know whether this subtle difference carries importance.)

I don't know if it is one or both of these points being referred to, or something else.

**Comment by Coburn** [comment]
We also ought to step back and look at the phrase objectively, and see what
it says. In plain English, it is,

"In fear and trembling, do the work that is your part of salvation, because
God is the one who is at work in you, so that you want and you do the
things that are desirable."

I am not sure from the OP what the exact issue is, but I am sure it will
help to get down to basics. Plus this is a challenging, encouraging verse!

Thanks,

Coby

**Comment by vonlein** [comment]
Greek scholar David Bentley Hart’s extremely literal translation of the original language into English is as follows (including verse 12 since it’s being included below):

12: Thus, my beloved ones, just as you have always been obedient (not as if only in my presence, but a great deal more so in my absence), work out your own salvation in reverence and trembling,

13: For it is God who is making active within you both the willing and the working of that which is dearly desirable.

New Testament scholar N.T. Wright’s less-literal/more-English take is:

12: So, my dear people: you always did what I said, so please now carry on in the same way, not just as though I was there with you, but much more because I’m not! Your task now is to work at bringing about your own salvation; and naturally you’ll be taking this with utter seriousness. 

13: After all, God himself is the one who’s at work among you, who provides both the will and the energy to enable you to do what pleases him.

An assertion about “the plain English” of a text in 2,000 year old Greek is never something you’ll find any two scholars completely agreeing on :)

**Comment by Coburn** [comment]
Different people see things differently. That is the nature of language. I
was just trying to put the same words to a different tune, and hopefully
help you guys to a) think outside the box and b) decide if this is an issue
worth resolving.

**Comment by openenglishbible** [comment]
This is, as Coburn says, a challenging and interesting verse. However I'm not sure that the existing wording is 'wrong enough' to warrant a change. To put it another way, to justify a change we need something manifestly better, not just different.

To throw some other translations into the mix:

NLT: For God is working in you, giving you the desire and the power to do what pleases him.
NET: for the one bringing forth in you both the desire and the effort—for the sake of his good pleasure—is God
REB: for it is God who works in you, inspiring both the will and the deed, for his own chosen purpose

I think the change here for me would be if anything to fix 'to will' which reads to me as being slightly archaic and/or higher registry than desired - maybe 'to want' to retain the alliteration?

**Comment by Coburn** [comment]
Just a bit of a comment here.

The NLT is the best of the three in terms of readability and good style.
But it implies that God gives us something called "power." I do not believe
that is warranted by this particular verse, and I do not believe it is the
implication here. I think it just means "ability."

The NET as usual is obscure and confusing.

The REB sounds awkward. We should not sound awkward.

Just those few thoughts. Thanks.

Coby

**Comment by Coburn** [comment]
It sounds good to me to say that "God is at work in us to motivate and
encourage us to do good works." That is not a translation, just a
paraphrase. But isn't it a wonderful thought? Phrasing it in that way gives
you insight into the motions of God's Spirit in your psyche in everyday
things. It makes me thankful and it does encourage and motivate me. Thanks
for bringing this verse to mind.
